libs/IPSymconExtension.js

```diff
@@ -138,7 +143,8 @@ class IPSymconExtension {
                 }
                 break;
             default:
```

Comment:
Für den aktuellen master branch kann einfach der default Teil entfallen, um die Fehler in Z2M zu entfernen.